ONBOARDING.md

```diff
@@ -31,7 +35,7 @@ This is very much a committee effort, and we are moving quickly enough that we h
 * Vivian Cromwell ([viviancromwell](https://github.com/viviancromwell)) - all things Design
 * Hunter Spinks ([hspinks](https://github.com/hspinks)) - all things Engineering
 
-# Getting Started
+# Building and Running
 
 Please review the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/CONTRIBUTING.md) before making any contributions to the code. Note that you'll need to add yourself to the [LICENSE](LICENSE) contributor list on your first commit.
```

Comment:
Let's remove the LICENSE instruction here as the "rules" source of truth is CONTRIBUTING
```suggestion
Please review the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/CONTRIBUTING.md) before making any contributions to the code.
```